packages/jaeger-ui/.depcheckrc.json

```diff
@@ -0,0 +1,12 @@
+{
+  "ignores": [
+    "@babel/preset-env",
+    "@babel/preset-react",
+    "@babel/preset-typescript",
+    "@types/react-dom",
+    "@types/react-router-redux",
+    "babel-plugin-inline-react-svg",
+    "jest-environment-jsdom"
+  ],
+  "skip-missing": true
```

Comment:
Missing deps would anyway be caught while linting because of no imports ts error.

Currently, we have the following as missing deps
```js
Missing dependencies
* react-is: ./src/utils/ReactShallowRenderer.test.js // Already installed as it is a dep for react-test-renderer (The warning suggests adding it as an explicit package inside package.json
* react-select: ./src/components/common/VirtSelect.tsx // same reason
* react-virtualized: ./src/components/common/VirtSelect.tsx // same reason
* _: ./build/static/index-21cea8b1.js // it is a build file
```

Reply:
Agreed! Would install them explicitly. But, would still need to skip the `build` directory for checking the deps (As in case of `* _: ./build/static/index-21cea8b1.js`)